tsuba: download the entire file before handing it off to parquet

tylershunt opened the pull request:
Chasing down a bug, In the full read case it shouldn't matter that much
anyway but we do lose some opportunity to overlap work this way.

Signed-off-by: Tyler Hunt <thunt@katanagraph.com>

ddn0 commented:
> (Not sure whether this flag is really needed to use llvm-10: -DLLVM_DIR="$(llvm-config-10 --cmakedir)")

Yeah, it is only needed if you want to specify the location of the llvm support libraries directly if for instance you wanted to compile with clang++-11 but use the support libraries from llvm-10. I think given the original issue about how clang, libllvm, gcc, etc. were getting all mixed up, I think giving instructions where all the locations are explicitly configurable should be helpful.

Commit:
tsuba: download the entire file before handing it off to parquet

Chasing down a bug, In the full read case it shouldn't matter that much
anyway but we do lose some opportunity to overlap work this way.

Signed-off-by: Tyler Hunt <thunt@katanagraph.com>

tylershunt commented:
This has been ruled out as a source of the current problem.
